hbase-protocol-shaded/src/main/protobuf/Backup.proto

```diff
@@ -54,11 +54,12 @@ message TableServerTimestamp {
 
 /**
  * Structure keeps relevant info for backup restore session
+ * backup_root_dir is deprecated because we no longer store root dirs on disk HBASE-28882
  */
 message BackupImage {
   optional string backup_id = 1;
   optional BackupType backup_type = 2;
-  optional string backup_root_dir = 3;
+  optional string backup_root_dir = 3 [deprecated = true];
```

Comment:
I'm not sure how cautiously we should treat the removal of this field. Deprecation and documentation seems like the safe approach, but this feature is in beta so maybe we can just remove it.